src/global/exception/common/global-exception.filter.ts

```
@@ -0,0 +1,76 @@
+import {
+    ExceptionFilter,
+    Catch,
+    ArgumentsHost,
+    HttpException,
+    HttpStatus,
+} from '@nestjs/common';
+import { Request, Response } from 'express';
+import { CustomWinstonLogger } from '../../logger/winston.logger';
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)